api/views/recordings_views.py

```diff
@@ -73,7 +75,7 @@ def post(self, request):
 
 @extend_schema(tags=["Recordings"])
 class UpdateRecordingTitleView(APIView):
-    permission_classes = [IsAuthenticated]
+    permission_classes = [IsAuthenticated, IsRecordingOwner]
```

Comment:
Is `IsAuthenticated` redundant here?

Reply:
You're correct, it is redundant. I'll fix that correct that.